documentation/frontend/reference/i18n.md

```diff
@@ -0,0 +1,75 @@
+# Internationalisation
+
+The frontend uses Nuxt's [i18n module](https://i18n.nuxtjs.org/) to support
+[internationalisation](https://developer.mozilla.org/en-US/docs/Glossary/I18N),
```

Comment:
Nit: should we use the British or American spelling? I'm much more used to the version with `z` (internationalization), but I know that we have several maintainers that are much more used to the British spelling.

Reply:
Updated in 4ffcee5.